.mailmap

```diff
@@ -29,7 +29,7 @@ Martin Nonnenmacher <martin.nonnenmacher@bosch.io> <mnonnenmacher@users.noreply.
 Mikko Murto <mikko.murto@doubleopen.org> <mikko.murto@hhpartners.fi>
 Mo Khan <mo.khan@gmail.com>
 Maximilian Huber <maximilian.huber@tngtech.com> <gh@maxhbr.de>
-Nicolas Nobelis <nicolas.nobelis@bosch.com> <nicolas.nobelis@bosch.io>
+Nicolas Nobelis <nicolas.nobelis@bosch.com> <nicolas.nobelis@bosch.com>
```

Comment:
There should be no need to list your `.com` email address twice, or? Both Bosch addresses already get consolidated to `Nicolas Nobelis <nicolas.nobelis@bosch.com>`, so it should be good as-is.

Reply:
Oh sorry Monday morning...

The point is the .io address should not be used anymore. Should I remove the mail or leave as is as you said ?

Reply:
Is is important regarding past commits ?

Reply:
> Is is important regarding past commits ?

Yes, so we should keep it.